src/Parallel/centrality/betweenness.jl

```diff
@@ -10,7 +10,7 @@ function betweenness_centrality(g::AbstractGraph,
 
     # Parallel reduction
 
-    betweenness = @distributed (+) for s in vs
+    betweenness = @distributed (+) for s in Int.(vs)
```

Comment:
This causes a dependence on `degree(g, s)` accepting a generic `Integer` for an AbstractGraph type that may be a specific type of integer. I'm not sure this is the best approach.

Reply:
Yes, and after https://github.com/JuliaLang/julia/pull/28651 merges then this workaround isn't necessary.

Reply:
I think what I'd rather do in the meantime (until the merged version is tagged) is to define
```
function splitrange(N::Integer, np::Integer) = splitrange(Int.(N), np)
```
locally. That way when the new version is tagged all we have to do is remove a line instead of remembering where to edit.